.eslintrc

```diff
@@ -34,6 +34,12 @@
   ],
   "settings": {
     "import/resolver": "webpack",
-    "import/extensions": [".js", ".jsx"]
+    "import/extensions": [".js", ".jsx"],
+    "react": {
+      "version": "15.4" // React version, default to the latest React stable release
+    }
+  },
+  "rules":{
+    "react/no-deprecated":"error"
```

Comment:
Not sure if we want error here, also we should move this to our eslint plugin config

Reply:
https://github.com/getsentry/eslint-config-sentry/blob/master/packages/eslint-config-sentry-react/rules/react.js#L48 this implies it was already set?